src/main/java/uk/gov/hmcts/reform/pcqloader/helper/PayloadValidationHelper.java

```diff
@@ -49,10 +50,14 @@ public void validateDisabilityImpact(PcqAnswers answers) {
     }
 
     public void validateLanguageLevel(PcqAnswers answers) {
-        //If user has selected the main language as English/Welsh or Prefer Not to Say then
-        //set the language level to null.
-        if (answers.getEnglishLanguageLevel() != null && answers.getLanguageMain() != null
-            && (answers.getLanguageMain() == 1 || answers.getLanguageMain() == 0)) {
+        //If user has selected the main language as English, Welsh, English or Welsh (legacy language option)
+        // or Prefer Not to Say
+        // then set the language level to null.
```

Comment:
Something doesn't make sense here.. Without context it sounds like no matter what user did, set language level to null. If they selected something, set to null, if not selected, I would naturally think it is null by default..